src/elements/content-sidebar/activity-feed/version/__tests__/Version.test.js

```diff
@@ -68,7 +71,7 @@ describe('elements/content-sidebar/ActivityFeed/version/Version', () => {
         ${defaultUser}      | ${defaultUser.name}
         ${restoreUser}      | ${restoreUser.name}
         ${trashedUser}      | ${trashedUser.name}
-        ${PLACEHOLDER_USER} | ${''}
+        ${PLACEHOLDER_USER} | ${priorCollaborator}
```

Comment:
this should be a hardcoded string, if someone changes the message unintentionally, the test will still pass